frontend/src/components/CreateIdeal.js

```diff
@@ -0,0 +1,145 @@
+import React from 'react';
+import {  useState } from 'react';
+import { useNavigate, useLocation } from 'react-router-dom';
+
+
+function CreateIdeal() {
```

Comment:
you could do inline export and save 1 line :) less lines / clutter is better overall, but don't overdo it 

```suggestion
export default function CreateIdeal() {
```